HeterogeneousCore/SonicTriton/scripts/cmsTriton

```diff
@@ -346,10 +346,22 @@ edit_model(){
 	IFS='/' read -ra ADDR <<< "$MODELNAME"
 	CONFIG=$TMPDIR/$LOCALMODELREPO/${ADDR[-1]}/config.pbtxt
 	
+	MODELLOC=$(echo ""${ADDR[@]:0:${#ADDR[@]}-1} | sed "s/ /\//g")
+
 	PLATFORM=$(grep -m 1 "^platform:" "$CONFIG")
-	
-	cat <<EOF >> $CONFIG
 
+	if [[ $PLATFORM == *"ensemble"* ]]; then
+		#recurse over submodels of ensemble model
+		SUBNAME=$(grep "model_name:" "$CONFIG")
+		for s in ${SUBNAME}; do
+			if ! [[ $s == *"model_name"* ]]; then
+				SUBMODEL=$(echo ${MODELLOC}/$s | tr -d '"')
```

Comment:
I think this can be simplified using sed:
```bash
SUBNAME=$(grep "model_name:" "$CONFIG" | sed 's/model_name://; s/"//g')
for SUBMODEL in ${SUBNAME}; do
```

Reply:
Ah thank you for the suggestion; implemented in latest commit